doc/release-notes.md

```diff
@@ -28,22 +35,378 @@ reindex or re-sync the whole chain.
+  
+dash-cli Changes
+----------------
+
+- A new `-rpcwaittimeout` argument to `dash-cli` sets the timeout
+  in seconds to use with `-rpcwait`. If the timeout expires,
+  `dash-cli` will report a failure. (dash#5923)
+- The `connections` field of `dash-cli -getinfo` is expanded to return a JSON
+  object with `in`, `out` and `total` numbers of peer connections and `mn_in`,
+  `mn_out` and `mn_total` numbers of verified mn connections. It previously
+  returned a single integer value for the total number of peer connections. (dash#5823)
+- Update `-getinfo` to return data in a user-friendly format that also reduces vertical space.
+- CLI `-addrinfo` now returns a single field for the number of `onion` addresses
+  known to the node instead of separate `torv2` and `torv3` fields, as support
+  for Tor V2 addresses was removed from Dash Core in 18.0.
+
+dash-wallet changes
+-------------------
+This release introduces several improvements and new features to the `dash-wallet` tool, making it more versatile and
+user-friendly for managing Dash wallets.
+
+Wallets created with the `dash-wallet` tool will now utilize the `FEATURE_LATEST` version of wallet which is the HD
+(Hierarchical Deterministic) wallets with HD chain inside.
+
+- new command line argument `-usehd` which let to create non-Hierarchical Deterministic (non-HD) wallets with the
+  `wallettool` for compatibility reasons since default version is bumped to HD version
+- new command line argument `-descriptors` to enable _experimental_ support of Descriptor wallets. It lets users to
+  create descriptor wallets directly from the command line. This change aims to align the command-line interface with
+  the `createwallet` RPC, promoting the use of descriptor wallets which offer a more robust and flexible way to manage
+  wallet addresses and keys.
+- new commands `dump` and `createfromdump` have been added, enhancing the wallet's storage migration capabilities. The
+  `dump` command allows for exporting every key-value pair from the wallet as comma-separated hex values, facilitating a
+  storage agnostic dump. Meanwhile, the `createfromdump` command enables the creation of a new wallet file using the
+  records specified in a dump file. These commands are similar to BDB's `db_dump` and `db_load` tools and are crucial 
+  for manual wallet file construction for testing or migration purposes.
+
+
+P2P and network changes
+-----------------------
 
-A deadlock caused nodes to become non-responsive and RPC to report "Work depth queue exceeded".
-Thanks to Konstantin Akimov (knst) who discovered the cause. This previously caused masternodes to become PoSe banned.
+- The protocol version has been bumped to 70232. This should be identical in behavior to 70231.
+- With I2P connections, a new, transient address is used for each outbound
+  connection if `-i2pacceptincoming=0`.
+- A dashd node will no longer rumour addresses to inbound peers by default.
```

Comment:
I don't know what this means
```suggestion
- A dashd node will no longer rumour addresses to inbound peers by default.
```

Reply:
It basically means as I understand, we don't immediately send a peer the nodes we know about, they must first request a node

Reply:
Okay, made a suggestion